aiohasupervisor/models/addons.py

```diff
@@ -99,6 +99,16 @@ class AddonState(StrEnum):
     ERROR = "error"
 
 
+class AddonStartup(StrEnum):
+    """AddonStartup type."""
+
+    INITIALIZE = "initialize"
+    SYSTEM = "system"
+    SERVICES = "services"
+    APPLICATION = "application"
+    ONCE = "once"
```

Comment:
Does this attribute order carry some meaning? Otherwise I'd suggest ordering it 🔡.